Makefile

```diff
@@ -1,6 +1,9 @@
 .DEFAULT_GOAL = test
 .PHONY: FORCE
 
+# Enable Go module support across all commands.
+export GO111MODULE = on
```

Comment:
Is this required? Go 1.13 has this on by default.

Reply:
@tariq1890 Go 1.13 defaults it to `auto` which implies module-awareness by default (see: https://golang.org/doc/go1.13#modules). However, the primary reason for this change is to consolidate all of the places it was being set and to force consistent module `on` behavior for both Go 1.12 and 1.13 builds in CI (see: https://github.com/golangci/golangci-lint/blob/master/.travis.yml#L2). FWIW, I also tested this change against Go 1.11 locally and it seemed to work fine. It may be possible to re-enable Go 1.11 in CI if there aren't any internal Go runtime/interface issues to be wary of. @jirfag Thoughts?